gammapy/estimators/flux_map.py

```diff
@@ -385,41 +400,41 @@ def sqrt_ts(self):
             sqrt(TS) map
         """
         if "sqrt_ts" in self._data:
-            return self._data["sqrt_ts"]
+            return self._filter_convergence_failure(self._data["sqrt_ts"])
         else:
             with np.errstate(invalid="ignore", divide="ignore"):
                 ts = np.clip(self.ts.data, 0, None)
                 data = np.where(self.norm > 0, np.sqrt(ts), -np.sqrt(ts))
-                return Map.from_geom(geom=self.geom, data=data)
+                return self._filter_convergence_failure(Map.from_geom(geom=self.geom, data=data))
```

Comment:
here the filtering has been already applied when doing `self.ts.data` no?

Reply:
very good point, thanks